Sources/StencilSwiftKit/Environment.swift

```diff
@@ -7,7 +7,7 @@
 import Stencil
 
 public extension Extension {
-  public func registerStencilSwiftExtensions() {
+  func registerStencilSwiftExtensions() {
```

Comment:
Why this change?

Reply:
most likely due to getting this compiler error if you try and mark the `func` as `public` when the extension is marked as `public`:

```
'public' modifier is redundant for instance method declared in a public extension
```

Reply:
Oh, good point! 👍